### PR TITLE
Spelling: Turn on/off, Bluetooth

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

**Comment by jeremypw** [comment]
Definitely an improvement but flagging for UX team for final sign off on text.

**Comment by comradekingu** [comment]
@jeremypw Can I be on the UX team (other than signing off on my commits)?

**Comment by jeremypw** [comment]
@comradekingu You need to contact @danrabbit I guess.

**Comment by danrabbit** [comment]
@cassidyjames might have opinions about this. Even though "turn off" and "turn on" sound more like conversational English, I wonder if "disable" and "enable" are more visually distinct. I think so far the precedent has been to use "disable" and "enable"

File: src/Widgets/DisplayWidget.vala [hunk 1]
```diff
@@ -70,7 +70,7 @@ public class BluetoothIndicator.Widgets.DisplayWidget : Gtk.Spinner {
 
         if (state) {
             style_context.remove_class ("disabled");
-            context = _("Middle-click to disable bluetooth");
+            context = _("Middle-click to turn off Bluetooth");
```
Comment (cassidyjames) [inline src/Widgets/DisplayWidget.vala:73]: This is idiomatic, but sounds better to me in English, at least. And you can easily replace `Bluetooth` with a pronoun and it still reads correctly, i.e. `Turn it off`.

```suggestion
            context = _("Middle-click to turn Bluetooth off");
```

File: src/Widgets/DisplayWidget.vala [hunk 2]
```diff
@@ -82,7 +82,7 @@ public class BluetoothIndicator.Widgets.DisplayWidget : Gtk.Spinner {
             style_context.remove_class ("paired");
             style_context.add_class ("disabled");
             description = _("Bluetooth is off");
-            context = _("Middle-click to enable bluetooth");
+            context = _("Middle-click to turn on Bluetooth");
```
Comment (cassidyjames) [inline src/Widgets/DisplayWidget.vala:85]: Same as above

```suggestion
            context = _("Middle-click to turn Bluetooth on");
```

**Review by cassidyjames — CHANGES_REQUESTED** [review]
Nice improvement, just a word order nitpick.

**Comment by cassidyjames** [comment]
@danrabbit ah I had loaded this tab before your comment and missed it. :) I'm mostly indifferent on "turn it off" vs "disable", though I do think turning off/on is how most people would say it. Enable/disable sounds more technical to me.

**Comment by comradekingu** [comment]
I am more strongly for "turn on/off", seeing as it in this case it is actually on or off (no power), and not just enabled or disabled.
The distinction is if something can be disabled, but still have power, or also need to be enabled in addition to turning on. Only then, and if that also makes sense to point out I favour "enable/disable".

**Review by cassidyjames — APPROVED** [review]

**Pull request merged**